projects/storefrontlib/src/lib/cms/facade/cms.service.ts

```diff
@@ -5,6 +5,7 @@ import { filter, tap, map, take } from 'rxjs/operators';
 import { select, Store } from '@ngrx/store';
 import { Page } from '../models/page.model';
 import { DefaultPageService } from '../services/default-page.service';
+import { Component } from '@spartacus/core';
```

Comment:
We have many kinds of cms components, e.g. cmsLinkComponent. These components should have different types. `Component` inside `@spartacus/core` is the base type of all cms components.

Reply:
I don't see any instances of any class extending `Component`